package.json

```diff
@@ -44,7 +44,6 @@
     "gulp-cssnano": "2.1.0",
     "gulp-eslint": "^1.0.0",
     "gulp-if": "^1.2.5",
-    "gulp-imagemin": "^2.3.0",
```

Comment:
Unrelated, but this is slow, and we shouldn't be using **gulp-imagemin** currently, per #908 -- upstream **gulp-imagemin** bug is still open and people are still reporting the issue.